<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ecatProb</class>
 <widget class="QWidget" name="ecatProb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ecatProb</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox QAbstractItemView::item { min-height: 40px; min-width: 60px; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,2">
   <property name="spacing">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>ProbeParam</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="3">
        <widget class="QLineEdit" name="lineEdit_probelPrm">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Probe1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_probel1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>0. OFF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1. Enable</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Probe2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="comboBox_probel2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>0. OFF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1. Enable</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_type1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>0. Signal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1. Continous</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QComboBox" name="comboBox_type2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>0. Signal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1. Continous</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Rising</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_rising1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>0. Switch off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1. Switch on</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Rising</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QComboBox" name="comboBox_rising2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>0. Switch off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1. Switch on</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Falling</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_falling1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>0. Switch off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1. Switch on</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Falling</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QComboBox" name="comboBox_falling2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>0. Switch off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1. Switch on</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_set">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>SetProbePrm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_get">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>GetProbeStatus</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>ProbeStatus</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_falling1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(255, 255, 255);
border:0;
</string>
       </property>
       <property name="text">
        <string>Falling</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>RisingValue</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_risingValue1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>RisingValue</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5" colspan="2">
      <widget class="QLineEdit" name="lineEdit_risingValue2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_fallingValue1">
       <property name="text">
        <string>FallingValue</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_fallingValue1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>FallingValue</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5" colspan="2">
      <widget class="QLineEdit" name="lineEdit_fallingValue2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_probelSts">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_probel1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(255, 255, 255);
border:0;
</string>
       </property>
       <property name="text">
        <string>Probel1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_rising1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(255, 255, 255);
border:0;
</string>
       </property>
       <property name="text">
        <string>Rising</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_probel2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(255, 255, 255);
border:0;
</string>
       </property>
       <property name="text">
        <string>Probel2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_rising2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(255, 255, 255);
border:0;
</string>
       </property>
       <property name="text">
        <string>Rising</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_falling2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(255, 255, 255);
border:0;
</string>
       </property>
       <property name="text">
        <string>Falling</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
